R/globals.R

```diff
@@ -141,8 +141,8 @@ globalVariables(c(
   "SI_UNIT_UPPER",
   "VAR_CHECK",
   "TERM",
-  "TERM_LEVEL",
-  "TERM_NAME",
+  "SRCVAR",
+  "TERMNAME",
   "TERM_NAME_ID",
```

Comment:
Looks like "TERM_ID"/"TERMID" was never needed?

Reply:
Not sure what you mean, should i update this file?